### PR TITLE
[Kotlin Cleanup] ReminderServiceTest

### PR DESCRIPTION
**Pull request by oyeraghib**
## Purpose / Description
[Kotlin Cleanup] ReminderServiceTest

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/anki/services/ReminderServiceTest.kt [hunk 1]
```diff
@@ -21,40 +21,39 @@ import android.content.Intent
...
 
 @RunWith(AndroidJUnit4::class)
-@KotlinCleanup("use matchers for size checks")
-@KotlinCleanup("`when` -> whenever()")
 class ReminderServiceTest : RobolectricTest() {
     @Test
     fun testReminderServiceNothingDue() {
         buildDefaultDeckReminders()
-        assertThat("No notifications exist", notificationManagerShadow.size() == 0)
+        assertThat("No notifications exist", notificationManagerShadow.size(), Matchers.equalTo(0))
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/anki/services/ReminderServiceTest.kt:38]: import `Matchers`.

See if you can use `Matchers.hasSize` instead

**Review by david-allison — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/test/java/com/ichi2/anki/services/ReminderServiceTest.kt (hunk 1 above)
Comment (oyeraghib) [reply AnkiDroid/src/test/java/com/ichi2/anki/services/ReminderServiceTest.kt:38]: using `hasSize` is causing

`Type mismatch.
Required:
(MutableCollection<out TypeVariable(E)!>..Collection<TypeVariable(E)!>?)`
`Found:
Int!`

Comment (david-allison) [reply AnkiDroid/src/test/java/com/ichi2/anki/services/ReminderServiceTest.kt:38]: Import `Matchers` and it's done then

Comment (oyeraghib) [reply AnkiDroid/src/test/java/com/ichi2/anki/services/ReminderServiceTest.kt]: Not sure but I guess the `hasSize` requires `MutableCollection` something. It still shows a mismatch :

`Type mismatch.
Required:
(MutableCollection<out TypeVariable(E)!>..Collection<TypeVariable(E)!>?)`
`Found:
ShadowNotificationManager!`

Comment (david-allison) [reply AnkiDroid/src/test/java/com/ichi2/anki/services/ReminderServiceTest.kt]: Sure, fine to leave it then

**Review by david-allison — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
:boom:

**Pull request merged**